api/sales_dashboard/forms.py

```diff
@@ -28,6 +33,29 @@ def save(self, organisation, commit=True):
         organisation.subscription.save()
 
 
+class StartTrialForm(forms.Form):
+    max_seats = forms.IntegerField()
+    max_api_calls = forms.IntegerField()
+
+    def save(self, organisation, commit=True):
+        organisation.subscription.max_seats = self.cleaned_data["max_seats"]
+        organisation.subscription.max_api_calls = self.cleaned_data["max_api_calls"]
+        organisation.subscription.subscription_id = TRIAL_SUBSCRIPTION_ID
+        organisation.subscription.customer_id = TRIAL_SUBSCRIPTION_ID
+        organisation.subscription.plan = "enterprise-saas-monthly-v2"
```

Comment:
Feels like we should accept a couple of different options here, right? Sometimes we want to give people a trial of scale up I think?

Reply:
I think it makes sense to offer them everything - it may lead to an upsell to Enteprise.